### ISSUE TITLE
Feature Request: Additional info added to inspect

### ISSUE THREAD

**Issue by cormachogan** [vmware/vic#1476]
**VIC Version:**

0.1 / master / 2832

Currently the VCH inspect output is as follows:

``` ts
./vic-machine-linux inspect  -t 'administrator@vsphere.local:VMware123!@10.27.51.103'  --compute-resource Mgmt --name VCH01
INFO[2016-07-13T14:20:34Z] ### Inspecting VCH ####
INFO[2016-07-13T14:20:34Z]
INFO[2016-07-13T14:20:34Z] VCH: /CNA-DC/host/Mgmt/Resources/VCH01
INFO[2016-07-13T14:20:34Z]
INFO[2016-07-13T14:20:34Z] Log server:
INFO[2016-07-13T14:20:34Z] https://10.27.51.38:2378
INFO[2016-07-13T14:20:34Z]
INFO[2016-07-13T14:20:34Z] DOCKER_HOST=10.27.51.38:2376
INFO[2016-07-13T14:20:34Z]
INFO[2016-07-13T14:20:34Z] Connect to docker:
INFO[2016-07-13T14:20:34Z] docker -H 10.27.51.38:2376 --tls info
INFO[2016-07-13T14:20:34Z] Completed successfully
root@photon-NaTv5i8IA [ /workspace/vic2832 ]#

```

I would like it to include the following information:
- image datastore
- container datastore (if different than default)
- volume store (if set)
- bridge network
- external network (if different than default)
- client network (if different than default)
- container network (if different than default)
- container network gw/dns/ip range
- iso images (if different than default)
- certificate info
- appliance cpu/memory info

**Comment by mdubya66** [comment]
Assigning to @emlin for inclusion in GA. @karthik-narayan @reddolan we should take a look at this one for gathering more input from the Onsite customers.

**Comment by reddolan** [comment]
I created a document of questions that we are looking to ask customers at the Onsite: https://docs.google.com/a/vmware.com/document/d/1jCGLSsv8hBihu2nwNO-yvt73YdDiU3unB51tUHAby60/edit?usp=sharing

**Comment by emlin** [comment]
@hickeng @mdubya66 @karthik-narayan Is there specific answer for vic-machine inspect command?

**Comment by emlin** [comment]
@hickeng Please help to confirm on this issue. thanks

**Comment by mdubya66** [comment]
Moving to Icebox per 9/20 Triage

**Comment by emlin** [comment]
This issue is addressed by #4618
